config/initializers/assets.rb

```diff
@@ -10,16 +10,17 @@
 # Precompile additional assets.
 # application.js, application.css, and all non-JS/CSS in app/assets folder are already added.
 # Rails.application.config.assets.precompile += %w( search.js )
-Rails.application.config.assets.precompile += %w( dependencies/html5shiv.min.js )
-Rails.application.config.assets.precompile += %w( dependencies/respond.min.js )
-Rails.application.config.assets.precompile += %w( dependencies/jquery.treetable.js )
-Rails.application.config.assets.precompile += %w( admin/admin.js )
-Rails.application.config.assets.precompile += %w( ie_lt9.js )
-Rails.application.config.assets.precompile += %w( search_form.js )
-Rails.application.config.assets.precompile += %w( areas.js )
-Rails.application.config.assets.precompile += %w( events.js )
-Rails.application.config.assets.precompile += %w( holders.js )
-Rails.application.config.assets.precompile += %w( users.js )
-Rails.application.config.assets.precompile += %w( organizations.js )
-Rails.application.config.assets.precompile += %w( admin.css )
-Rails.application.config.assets.precompile += %w( email.css )
+Rails.application.config.assets.precompile += ['dependencies/html5shiv.min.js']
```

Comment:
I do not understand this change!

Reply:
To add the new js, but now I deleted the file